corehq/apps/users/forms.py

```diff
@@ -1269,6 +1268,12 @@ def clean_search_string(self):
             raise forms.ValidationError(_("* and ? are not allowed"))
         return search_string
 
+    def clean_domains(self):
+        domains = self.data.getlist('domains[]', [self.domain])
```

Comment:
Interesting, I didn't know you could include the brackets here.

Reply:
I found that data contains "domains[]" exclusively before clicking submit and "domains" exclusively after clicking submit. I could not find anywhere _why_ it's that way, so I handle both for the pre-submit count_users function.